etcd-3.5.yaml

```diff
@@ -41,6 +41,9 @@ pipeline:
 subpackages:
   - name: etcd-3.5-bitnami-compat
     description: "compat package with bitnami/etcd image"
+    dependencies:
+      runtime:
+        - bash
```

Comment:
Was defined as a runtime dep at the image level - moving to package level, aligning with other similar examples.

Reply:
Thanks for moving these over